_docs/project-ideas.md

```diff
@@ -0,0 +1,25 @@
+Stuck for ideas? You're not alone. Even the creative geniuses of our time need places to start to get some inspiration. Here are some ideas to help you get started.
+
+## Make Something Retro
+- ?
+
+## Solve a Community Problem
+- transit app?
+- food ?
+- textbooks?
+- the stuff listed on '19 project-ideas
+
+## Make Something Helpful
+- tutor
+- adress eye problems
+- games to teach reading/spelling?
+- games to teach new language?
```

Comment:
Flesh these out; doesn't need to be anything detailed. See the section under `## Make Something Silly` for examples of formatting/level of detail